front-end/nuxt.config.ts

```diff
@@ -3,4 +3,8 @@ export default defineNuxtConfig({
   compatibilityDate: "2024-04-03",
   devtools: { enabled: true },
   modules: ["@nuxt/eslint"],
+  css: [
```

Comment:
Utilise SASS partials which should be imported to a `~/assets/styles/main.scss` file. Reconfigure Vite (as a build tool for Nuxt) to use that "root SASS module" to apply the global styles as follows:

```typescript
export default defineNuxtConfig({
  vite: {
    css: {
      preprocessorOptions: {
        scss: {
          additionalData: '@use "~/assets/styles/main.scss" as *;'
        }
      }
    }
  }
})
```